pkg/volume/cephfs/cephfs.go

```diff
@@ -301,13 +304,40 @@ func (cephfsVolume *cephfs) execMount(mountpoint string) error {
 	return nil
 }
 
-func getVolumeSource(spec *volume.Spec) (*v1.CephFSVolumeSource, bool, error) {
+func getVolumeSource(spec *volume.Spec) (mon []string, path, user, secretFile string, readOnly bool, err error) {
 	if spec.Volume != nil && spec.Volume.CephFS != nil {
-		return spec.Volume.CephFS, spec.Volume.CephFS.ReadOnly, nil
+		mon = spec.Volume.CephFS.Monitors
+		path = spec.Volume.CephFS.Path
+		user = spec.Volume.CephFS.User
+		secretFile = spec.Volume.CephFS.SecretFile
+		readOnly = spec.Volume.CephFS.ReadOnly
+		err = nil
+		return
```

Comment:
prefer explicit returns over bare returns to ensure you are setting and returning everything you think you are